projects/Mallard/src/services/__tests__/crashlytics.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { CrashlyticsService } from '../crashlytics'
+
+jest.mock('@react-native-firebase/crashlytics', () => ({
+    setCrashlyticsCollectionEnabled: jest.fn(() => {}),
+    recordError: jest.fn((err: Error) => {}),
+}))
+
+describe('crashlyticsService', () => {
+    const crashlytics = require('@react-native-firebase/crashlytics')
+    let crashlyticsService: CrashlyticsService
+
+    beforeEach(() => {
+        crashlyticsService = require('../crashlytics').crashlyticsService
+    })
+
+    it('should send handled exception', () => {
+        crashlyticsService.captureException(new Error())
```

Comment:
Having an issue with testing here would be good to pair on this.

error:
`crashlytics: is not a function`